development/docker-compose/docker-compose.yaml

```diff
@@ -244,32 +248,32 @@ services:
     links:
       - registry
 
-  product-catalog-task:
-    image: repo.backbase.com/backbase-stream-images/product-catalog-task:${STREAM_VERSION}
-    profiles:
-      - bootstrap
-    environment:
-      <<: *common-variables
-      eureka.client.register-with-eureka: false
-      spring.config.additional-location: file:/tmp/ingestion.yml
-    volumes:
-      - ../config/stream/ingestion.yml:/tmp/ingestion.yml
-    depends_on:
-      token-converter:
-        condition: service_healthy
-      arrangement-manager:
-        condition: service_healthy
-
-  legal-entity-bootstrap-task:
-    image: repo.backbase.com/backbase-stream-images/legal-entity-bootstrap-task:${STREAM_VERSION}
+  bootstrap-job:
+    image: bootstrap-job:${BOOTSTRAP_JOB_VERSION}
     profiles:
       - bootstrap
+    build:
+      context: ../images/bootstrap
+      secrets:
+        - mvnrepo
+      args:
+        JOB_VERSION: ${BOOTSTRAP_JOB_VERSION}
     environment:
       <<: *common-variables
-      eureka.client.register-with-eureka: false
-      spring.config.additional-location: file:/tmp/ingestion.yml
+      spring.autoconfigure.exclude: org.springframework.boot.autoconfigure.security.oauth2.client.reactive.ReactiveOAuth2ClientAutoConfiguration
+      backbase.communication.http.discoverable-access-token-service: true
+      backbase.stream.limits.worker.enabled: false
+      backbase.bootstrap.data.active-data-sets: local-backend-setup
+      backbase.bootstrap.data.legal-entity-prefix.enabled: false
+      backbase.bootstrap.ingestions.identity-users.enabled: false
+      backbase.bootstrap.ingestions.messages.enabled: false
+      backbase.bootstrap.ingestions.push-integration.enabled: false
+      backbase.bootstrap.ingestions.content-repository.enabled: false
+      backbase.bootstrap.ingestions.fido-applications.enabled: false
+      backbase.bootstrap.identity.baseUrl: http://backbase-identity:8080/auth
+      admin.baseUrl: http://backbase-identity:8080
     volumes:
-      - ../config/stream/ingestion.yml:/tmp/ingestion.yml
+      - ./data/:/opt/resources/
```

Comment:
why not move the LegalEntity.json and the products.json to the ./data folder then? it would be easier for the developer to play with them if they are already in the right place

Reply:
because the whole Idea was to use the default data in bootstrap-job. if we place the default data, again it needs to be maintained in case the structure changes.